File: C Programming/Assignments/Assignment 6/function with no return type and no para/Assignment 1/Q5/Q5.c
//Write a program to check whether a person is eligible to vote (age >= 18).

#include<stdio.h>

void checkAge();
void main(){
	
	checkAge();
	
}
	
void checkAge(){

	//variable initialization
	int age = 17;
	
	//applying condition
	if(age >= 18)
		printf("Eligible to vote");
	else
		printf("Not eligible to vote - Your age is %d years old\nAge must be at least 18 years old to be eligible.",age);

}

File: FirstBit Workspace/C Programming/Assignments/java_to_c_connectivity/Assignment 1/Q5/Q5.java
//Write a program to check whether a person is eligible to vote (age >= 18).

public class Q5 {
    public static void main(String[] args) {
        // variable initialization
        int age = 17;

        // applying condition
        if (age >= 18)
            System.out.println("Eligible to vote");
        else
            System.out.println("Not eligible to vote - Your age is " + age + " years old\nAge must be at least 18 years old to be eligible.");
    }
}

